.vale

```diff
@@ -0,0 +1,6 @@
+# Global settings (applied to every syntax)
+[*]
+# List of styles to load
+BasedOnStyles = vale
```

Comment:
The PlainLanguage style also looks really useful - looks like they moved it out of the core one. https://github.com/ValeLint/vale/tree/master/styles/PlainLanguage

Reply:
That does look good. I'll try enabling it to see what output we return.

Reply:
I enabled PlainLanguage, and most results were about slashes, and so there were quite a few false positives:

```
41:37  error    Use either 'or' or 'and' in     PlainLanguage.Slash
                        'alphagov/govuk'
```

I'm going to set that to `warning` only, but in the future I may look at increasing "wordiness" and "sentencelength" to `error` as these seem really useful from this style.